torch/_torch_docs.py

```diff
@@ -4530,6 +4530,11 @@ def merge_dicts(*dicts):
 
 Returns the median value of all elements in the :attr:`input` tensor.
 
+.. note::
+    For even-sized inputs, the smaller of the two middle elements in the sorted
```

Comment:
I'm not in love with saying "smaller." For example:

```
t = torch.tensor((-3, -5))
torch.median(t)
: tensor(-5)
```

Is -5 "smaller" than -3? I feel like that's debatable at best. What about using the term "lower"? That would be consistent with std::numeric_limits, for example.

"Middle elements" is also a little vague. I would appeal to the more formal definition of the median https://en.wikipedia.org/wiki/Median, say it may not be unique, and that when the median is not unique we pick the lower of the two medians in the input.